doc/source/users_guide/index.rst

```diff
@@ -51,110 +50,204 @@ This example shows how you can display a mesh:
         "solid_up:1:830",
         "solid_up:1:830-shadow",
     ]
-    mesh1.display("window-1")
+
+    mesh_object_1 = Mesh(solver=solver_session, show_edges=True, surfaces=mesh_surfaces_list)
+    mesh_window = GraphicsWindow(grid=(1, 2))
+    mesh_window.add_graphics(mesh1, position=(0, 0))
+
+    mesh_object_2 = Mesh(solver=solver_session, surfaces=mesh_surfaces_list)
+    mesh_object_2.show_edges = False
+
+    mesh_window.add_graphics(mesh2, position=(0, 1))
+    mesh_window.show()
+
+Display plane-surface
```

Comment:
I realize plane-surface is what is used in the code. However, wouldn't you use "plane surface" in a heading or sentence?